### ISSUE TITLE
Improve error message for non-compliant WCS keywords

### ISSUE THREAD

**Issue by astrofrog** [astropy/astropy#4389]
As mentioned by @embray in [this](http://stackoverflow.com/questions/34165313/unable-to-read-wcs-from-sdss-co-added-stripe-82-fits-image/34166219?noredirect=1#comment56090635_34166219) stackoverflow comment, the following warning is confusing:

```
WARNING: FITSFixedWarning: RADECSYS= 'ICRS ' 
RADECSYS is non-standard, use RADESYSa. [astropy.wcs.wcs]
```

because the user didn't do anything wrong in all likelihood. Maybe we can rephrase to:

```
WARNING: RADECSYS is non-standard, changing it to RADESYSa. [astropy.wcs.wcs]
```

or something like this? (to emphasize that it is getting fixed)

**Comment by embray** [comment]
Yeah, something along those lines.  I didn't realize it was being automatically fixed though.
